taca/cleanup/cleanup.py

```diff
@@ -0,0 +1,644 @@
+        list_to_delete.extend(get_closed_projects(projects, pcon, seconds))
+    elif site == "archive":
+        ##work flow for cleaning archive ##
+        runs = [ r for r in os.listdir(root_dir) if re.match(filesystem.RUN_RE,r) ]
+        for run in runs:
+            with filesystem.chdir(os.path.join(root_dir, run)):
+                ## Collect all project path from demultiplexed directories in the run folder
+                all_proj_path = glob(config_ppath)
+                all_proj_dict = {os.path.basename(pp).replace('Project_','').replace('__', '.'): pp for pp in all_proj_path}
+                closed_projects = get_closed_projects(all_proj_dict.keys(), pcon, seconds)
+                ## Only proceed cleaning the data for closed projects
+                for closed_proj in closed_projects:
+                    closed_proj_fq = glob("{}/*/*.fastq.gz".format(all_proj_dict[closed_proj]))
+                    list_to_delete.extend([os.path.join(run, pfile) for pfile in closed_proj_fq])
+                ## Remove the undetermined fastq files for NoIndex case always
+                undetermined_fastq_files = glob(config_npath)
+                ## Remove undeterminded fastq files for all index length if all project run in the FC is closed
+                if len(all_proj_dict.keys()) == len(closed_projects):
+                    undetermined_fastq_files = glob(config_upath)
+                list_to_delete.extend([os.path.join(run, ufile) for ufile in undetermined_fastq_files])
+
+    ## delete and log
+    for item in list_to_delete:
+        if dry_run:
+            logger.info('Will remove {} from {}'.format(item,root_dir))
+            continue
+        try:
+            to_remove = os.path.join(root_dir,item)
+            if os.path.isfile(to_remove):
+                os.remove(to_remove)
+            elif os.path.isdir(to_remove):
+                shutil.rmtree(to_remove)
+            logger.info('Removed {} from {}'.format(item,root_dir))
+            with open(log_file,'a') as to_log:
+                to_log.write("{}\t{}\n".format(to_remove,datetime.strftime(datetime.now(),'%Y-%m-%d %H:%M')))
+        except OSError:
+            logger.warn("Could not remove {} from {}".format(item,root_dir))
+            continue
+
+
+def cleanup_irma(days_fastq, days_analysis, only_fastq, only_analysis, dry_run=False):
+    """Remove fastq/analysis data for projects that have been closed more than given 
+    days (as days_fastq/days_analysis) from the given 'irma' cluster
+
+    :param int days_fastq: Days to consider to remove fastq files for project
+    :param int days_analysis: Days to consider to remove analysis data for project
+    :param bool only_fastq: Remove only fastq files for closed projects
+    :param bool only_analysis: Remove only analysis data for closed projects
+    :param bool dry_run: Will summarize what is going to be done without really doing it
+    """
+    try:
+        config = CONFIG['cleanup']['irma']
```

Comment:
would be great if you can make an example of this config, it will also help @sylvinite in deploying this